packages/shell-chrome/src/state.js

```diff
@@ -125,15 +130,22 @@ export default class State {
     }
 
     updateXdata() {
+        document.getElementById("header").setAttribute(
+            "x-data",
+            `{
+                version : '${this.version}',
```

Comment:
when we run this and update the attribute, what happens to the functions that were defined in x-data?

Reply:
Yikes, good point. Could inject it directly into the alpine instance?
```suggestion
        let headerData = document.getElementById("header").__x.$data;
        headerData.version = this.version.detected;
        headerData.latest = this.version.latest;
```

Reply:
yeah that could work

Reply:
Sorted, also updated the components to do the same 👍🏼